docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -93,21 +94,30 @@ else:
 
 **Inputs**:
 
-*   **1**: `data` -- input tensor of rank 3 or greater. Layout is `[N, C_INPUT, X1, ..., XD]`. *Required*.
+*   **1**: Input tensor of type *T1* and rank 3, 4 or 5. Layout is NCZYX (number of batches, number of channels, spatial axes Z, Y, X). *Required*.
```

Comment:
I would leave the shape layout as it was, so we know explicitly that C dim corresponds to I dim of filter.
`[N, C_INPUT, X1, ..., XD]` or `[N, C_INPUT, Z, Y, X]` if you prefer

Reply:
Yes, that is better solution to use existing layout.